docs/tutorials/issue-tracker/05-local-deployment.md

```diff
@@ -172,16 +172,15 @@ If we click on the `grafana` tile to see what the error is, it shows us the logs
 logger=provisioning t=2025-01-30T16:53:00.36954668Z level=error msg="Failed to provision plugins" error="app provisioning error: plugin not installed: \"issuetrackerproject-app\""
 ```
 
-So this is just something we didn't do before we started our environment. While we built the plugin, we want to also deploy it to our local cluster. The local cluster can only read from `local/mounted-files`, so we need it there. Simple enough fix, we run
+So this is just something we didn't do before starting our environment. When we build the plugin, we also need to deploy it to the local cluster. Because the cluster can only read from `local/mounted-files`, the plugin must be copied there. The fix is simple: run `make local/deploy_plugin` (in a separate terminal window).
+
```

Comment:
I got confused here as I first stopped the running `make local/up` and tried to run `make local/deploy_plugin` instead, which failed because the `make local/up` was not running.